packages/playground/src/components/deployment_data_dialog.vue

```diff
@@ -120,11 +131,50 @@ function copy() {
 }
 
 function getLabel(key: string): string {
-  if (props.environments !== false && typeof props.environments[key] === "string") {
-    return props.environments[key] as string;
+  if (props.environments === false) {
+    return key;
+  }
+
+  const env = props.environments[key];
+
+  if (typeof env === "string") {
+    return env;
+  }
+
+  if (typeof env === "object") {
+    return env.label || key;
   }
+
   return key;
 }
+
+function getValue(key: string) {
+  const value = contract.value.env[key];
+  const transform = (props.environments || ({} as any))[key]?.transform || _transform;
+  return transform(value);
+}
+
+function _transform(value: string): any {
+  const v = value.toLowerCase();
+  if (v === "true" || v === "false") {
+    return Boolean(v);
+  }
+  return value;
+}
+
+function getType(key: string): string {
+  if (props.environments === false) {
```

Comment:
I don't think this condition is necessary as it returns the same as the default.